pyspeckit/spectrum/tests/test_fitter.py

```diff
@@ -53,13 +54,19 @@ def test_fitter(self):
 
         # Do it again to make sure there are no behavioral changes the second time around
         self.sp.specfit(fittype='gaussian', guesses=(-1, 0, 0.5),
-                        limitedmin=(False,False,True), minpars=(0,0,1e-10))
+                        limitedmin=(False,False,True), minpars=(0,0,1e-10),
+                        use_lmfit=use_lmfit)
         assert self.sp.specfit.parinfo.limited == [(False,False),(False,False),(True,False)]
         assert self.sp.specfit.parinfo.limits == [(0,0),(0,0),(1e-10,0)]
         np.testing.assert_almost_equal(self.sp.specfit.parinfo[0].value, -1)
         np.testing.assert_almost_equal(self.sp.specfit.parinfo[1].value,  0)
         np.testing.assert_almost_equal(self.sp.specfit.parinfo[2].value,  1)
 
+    def test_lmfitter(self):
+        """
+        Regression test for #223: make sure lmfitter returns proper pars
+        """
+        self.test_fitter(use_lmfit=True)
```

Comment:
Not really sure if this test is being executed, probably needs updating tests...